### DISCUSSION TITLE
Birt 4.9 Runtime gives error when deployed

### DISCUSSION THREAD

**Discussion by sujatha-crown** [eclipse-birt/birt#1048] [Q&A] [answered]
Hello,
I downloaded the BIRT RUNTIME from https://download.eclipse.org/birt/downloads/drops/R-R1-4.9.0-202203161719/birt-runtime-4.9.0-20220315.zip. When we deployed BIRT war, it gave us error, after carefully examination, we found inside birt.war\WEB-INF\lib\javax.xml.rpc_1.1.2.v202203150031.jar file there was javax.xml.rpc-api-1.1.2.jar inside the jar file. When we removed and re-deployed the Runtime worked correctly. 
My question is 
a) are you aware of this issue already?
b) do you have new version release of Birt Runtime
Thanks

**Comment by wimjongman** [comment, marked as answer]
Please try the snapshot area and report back if this worked.

https://download.eclipse.org/birt/downloads/drops/snapshot/

**Reply by sujatha-crown** [reply]
Thank you! will try and let you know the feedback.

**Reply by sujatha-crown** [reply]
Thank you very much for your help it worked correctly. I appreciate all your help!

**Reply by sujatha-crown** [reply]
Is this URL permanent or will it change or be removed after a certain time?

**Reply by wimjongman** [reply]
It is permanent but it will change daily and it is not guaranteed to be stable. 

We are in the process of releasing BIRT 4.10. It will be released on the first of November 2022.

**Reply by sujatha-crown** [reply]
Thank you! will wait for the November realease
